CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ Types of changes are:
 
 ## [Unreleased]
 
+- Implement plugin system for commands
```

Comment:
Please add header to the change to show what type of change it is (Added, Changed, etc...) This affects the version number when releasing.

Reply:
[2a1b3f0](https://github.com/radeklat/delfino/pull/32/commits/2a1b3f0100b37ddd1a7ab6840a016149875cbac9)